ddb_cp.module

```diff
@@ -125,6 +125,8 @@ function ddb_cp_get_test_result($site = NULL, $number = NULL) {
   else {
     return MENU_NOT_FOUND;
   }
+
+  // @TODO: This function never returns any XML data only page not found error code?
```

Comment:
How?  As the content inside the if statement do not have an return statemen? The only return statement in the function returns MENU_NOT_FOUND, which is the number "2".